### DISCUSSION TITLE
Problems while manipulating DISCON.IN files

### DISCUSSION THREAD

**Discussion by aravindvenkatachalapathy** [OpenFAST/python-toolbox#77] [General]
Hello,
I am trying to manipulate DISCON.IN files written by ROSCO using FASTInputFile . I think the character '!' is not getting recognized.  What can be the possible solution for this?
![image](https://github.com/OpenFAST/python-toolbox/assets/43172847/f011809f-6a25-49fb-9156-90a08ed6a84e)

**Comment by ebranlard** [comment]
pyFAST has been renamed to `openfast_toolbox` and migrated here: https://github.com/OpenFAST/openfast_toolbox . 

Please continue the discussion in this new repository. Sorry for the inconvenience.
